src/D2L.Bmx/PrintHandler.cs

```diff
@@ -43,9 +43,9 @@ AwsCredsCreator awsCreds
 
 	private static void PrintBash( AwsCredentials credentials ) {
 		Console.WriteLine( $"""
-			export AWS_SESSION_TOKEN='{credentials.SessionToken}'
-			export AWS_ACCESS_KEY_ID='{credentials.AccessKeyId}'
-			export AWS_SECRET_ACCESS_KEY='{credentials.SecretAccessKey}'
+			export AWS_SESSION_TOKEN={credentials.SessionToken}
+			export AWS_ACCESS_KEY_ID={credentials.AccessKeyId}
+			export AWS_SECRET_ACCESS_KEY={credentials.SecretAccessKey}
```

Comment:
These single quotes ended up in the variable values themselves, resulting in wrong creds.
The distinction between the shell's preprocessing of commands and the actual values can be subtle...